### PR TITLE
Task revamp

### PR DESCRIPTION
**Pull request by SijiaDavis**
* Expose the `ll-options-max-height` css custom property so we can set the max height of the opened dropdown list
* remove a weird scroll tracking behavior 
* add `name` attribute to support form submission

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ This is of particular requirement when you hope to manage Object based value tha
 | `--ll-select-arrow-size-factor` | Number of padding widths to make available for the select arrow | `3` |
 | `--ll-select-color` | Text color for the `<ll-select/>` | `inherit` |
 | `--ll-select-color-background` | Background color for the `<ll-select/>` | `#fff` |
+| `--ll-options-max-height` | Max height for the options rendered inside the `<ll-select/>` | `none` |
```
Comment (edwinthinks) [inline README.md]:
```suggestion
| `--ll-options-max-height` | Max height for the dropdown menu rendered inside the `<ll-select/>` | `none` |
```

File: ll-select.js [hunk 2]
```diff
@@ -418,19 +419,6 @@ class Select extends LitElement {
         break;
     }
   }
-  /**
-   * Manage the location of the dropdown while the options are open
-   */
-  _scrollTracker() {
```
Comment (francomendoza) [inline ll-select.js]: how does this affect other components using the select? like the classroom options?

Comment (SijiaDavis) [reply ll-select.js]: @francomendoza it doesn't, that option doesn't have a long enough list for the scroll behavior to happen

**Comment by SijiaDavis** [comment]
@edwinthinks if the `ll-select` contains a long list of dropdowns, when you scroll down the list of dropdown the `select` component actually scrolls up, so it almost looks like the select is floating on top of everything else. I suppose if we tweak the scrolling speed in a certain way it can be a cool effect, but currently, it just looks like a mess on the task description page.

File: ll-select.js (hunk 2 above)
Comment (SijiaDavis) [reply ll-select.js]: and because it's currently only used in the classroom, I think it's safe to remove this behavior.

Comment (francomendoza) [reply ll-select.js]: is this behavior no longer necessary because of potential max-height?

Comment (SijiaDavis) [reply ll-select.js]: @francomendoza this behavior doesn't work 100% if the options get long, the "scrolling up / y transform to the top" of the select doesn't happen in a rate quick enough to counter the scrolling down motion when the options are very long. I also asked Hayashi to see if he has any insights into this behavior, he said he remembers having a conversation about it with WJ, but he doesn't remember the details.

but yeah, having the `max-height` also get rid of the need for this behavior

**Comment by SijiaDavis** [comment]
@francomendoza @edwinthinks let me know if there's anything else that needs changing before I merge it in and do a release

File: ll-select.js (hunk 2 above)
Comment (francomendoza) [reply ll-select.js]: ok I'm mostly trying to determine if we need to set a default max-height.  So if this behavior previously handled long lists, that means we did not code in that behavior into usages of the select in other places.  So we should either define max-height where ll-selects are used, or we can just set a default to handle those cases... thoughts?

Comment (SijiaDavis) [reply ll-select.js]: sounds good, I will set a default on it instead of `none`

File: ll-select-styles.js [hunk 3]
```diff
@@ -100,6 +100,8 @@ export const style = html`
     border: 2px solid;
     border-radius: var(--ll-select-border-radius-local);
     width: var(--ll-select-width-local);
+    max-height: var(--ll-options-max-height, none);
```
Comment (SijiaDavis) [inline ll-select-styles.js]:
```suggestion
    max-height: var(--ll-options-max-height, 200px);
```

File: README.md [hunk 4]
```diff
@@ -49,6 +49,7 @@ This is of particular requirement when you hope to manage Object based value tha
 | `--ll-select-arrow-size-factor` | Number of padding widths to make available for the select arrow | `3` |
 | `--ll-select-color` | Text color for the `<ll-select/>` | `inherit` |
 | `--ll-select-color-background` | Background color for the `<ll-select/>` | `#fff` |
+| `--ll-options-max-height` | Max height for the dropdown menu rendered inside the `<ll-select/>` | `none` |
```
Comment (SijiaDavis) [inline README.md]:
```suggestion
| `--ll-options-max-height` | Max height for the dropdown menu rendered inside the `<ll-select/>` | `200px` |
```

**Pull request merged**